pir/cpp/server.cpp

```diff
@@ -58,7 +60,7 @@ StatusOr<Response> PIRServer::ProcessRequest(
                    SEALDeserialize<GaloisKeys>(context_->SEALContext(),
                                                request_proto.galois_keys()));
 
-  const auto dimensions = context_->Parameters()->Dimensions();
+  const auto dimensions = context_->Params()->dimensions();
   const size_t dim_sum = std::accumulate(dimensions.begin(), dimensions.end(),
```

Comment:
Since we seem to use this in more than one place, perhaps this should be in Context? WDYT?
(And yes, I probably should have done that in the previous PR)